lib/active_merchant/billing/gateways/stripe.rb

```diff
@@ -365,6 +367,21 @@ def add_address(post, options)
         end
       end
 
+      def add_statement_address(post, options)
+        statement_address = options[:statement_address]
+
+        return unless statement_address && statement_address[:address1] &&
+          statement_address[:city] &&
+          statement_address[:zip] &&
+          statement_address[:state]
+        post[:statement_address] = {}
+        post[:statement_address][:line1] = statement_address[:address1]
+        post[:statement_address][:line2] = statement_address[:address2]
+        post[:statement_address][:city] = statement_address[:city]
+        post[:statement_address][:postal_code] = statement_address[:zip]
```

Comment:
What if I pass in a `statement_address[:postal_code]` value instead of `:zip`? Should this accept either one?